src/main/java/net/mcreator/ui/minecraft/EntityListField.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2024, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.minecraft;
+
+import net.mcreator.element.parts.EntityEntry;
+import net.mcreator.minecraft.ElementUtil;
+import net.mcreator.ui.MCreator;
+import net.mcreator.ui.dialogs.DataListSelectorDialog;
+import net.mcreator.ui.init.L10N;
+
+import java.util.List;
+
+public class EntityListField extends SpawnableEntityListField {

```

Comment:
It is a bit anti pattern to extend more specific class into less specific one. To me it also appears this is not needed since one will most likely want to only apply attributes to SpawnableEntityListField

Reply:
I need non-spawnable entities like LivingEntity to be selectable too

Reply:
I see, but still make this extend JItemListField<EntityEntry> as this is plain different selector and also likely won't operate on tags so getTagsToAdd override may not be needed, also not that many constructor types